pkg/cmd/pulumi/import.go

```diff
@@ -496,37 +501,39 @@ func newImportCmd() *cobra.Command {
 				Scopes:             cancellationScopes,
 			}, imports)
 
-			deployment, err := getCurrentDeploymentForStack(s)
-			if err != nil {
-				return result.FromError(err)
-			}
+			if generateCode {
```

Comment:
Minor nit: maybe pull the body of this `if` out into its own function? Normally I'd suggest using an early return here to be more idiomatic and avoid the rightward movement, but that isn't convenient with the `res != nil` bits later.

Reply:
I'd generally lean towards keeping the code inline here, we're just hitting 120 cols which doesn't seem unreasonable.